### PR TITLE
Pwa 6

### PR DESCRIPTION
**Pull request by sznowicki**
# Pull Request Template

## Description

- PWA6 migration
- Bring back placeholders on Sliders
- GMD/IOS styling fixes

## Type of change

- [x] Bug fix
- [x] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,5 +35,6 @@ First version of the extension.
 Initial commmit.
 
 [Unreleased]: https://github.com/shopgate/ext-upselling/compare/v1.1.0...HEAD
```
Comment (aaron-martin) [inline CHANGELOG.md]: Unreleased link should be updated to https://github.com/shopgate/ext-upselling/compare/v2.0.0...HEAD

File: frontend/components/PDPSlider/index.jsx [hunk 2]
```diff
@@ -62,8 +63,8 @@ class PDPSlider extends Component {
  * @param {Object} state State.
  * @returns {Object}
```
Comment (aaron-martin) [inline frontend/components/PDPSlider/index.jsx:65]: jsDoc missing props param

**Review by aaron-martin — CHANGES_REQUESTED** [review]
I found two small issues. Sorry to be so picky.

File: frontend/components/PDPSlider/index.jsx (hunk 2 above)
Comment (sznowicki) [reply frontend/components/PDPSlider/index.jsx:65]: done

File: CHANGELOG.md (hunk 1 above)
Comment (sznowicki) [reply CHANGELOG.md]: Done.

**Comment by sznowicki** [comment]
> I found two small issues. Sorry to be so picky.

No need to be sorry. That's the review process ;)

File: frontend/components/Slider/index.jsx [hunk 3]
```diff
@@ -36,35 +36,72 @@ class Slider extends Component {
...
+    if (this.fatal || !this.props.productIds.length) {
       return null;
     }
 
+    let availableProductIds = this.props.productIds;
+    if (this.state.allowPlaceholders === false) {
+      availableProductIds = this.props.productIds.filter(id => this.props.products[id]);
+
+      if (availableProductIds.length === 0) {
+        // Should never happen in real life. But it after timeout, there's still not even one
```
Comment (Carsten89) [inline frontend/components/Slider/index.jsx]:
```suggestion
        // Should never happen in real life. But if after timeout, there's still not even one
```

**Pull request merged**